nodejs/week1/warmup.js

```diff
@@ -0,0 +1,32 @@
+class Circle {
+    constructor(radius) {
+        this.radius = radius;
+    }
+
+    getDiameter() {
+        this.diameter = 2 * this.radius;
+        console.log(`The diameter is ${this.diameter}`);
+    }
+
+    getCircumference() {
+
+        this.circumference = 2 * Math.PI * this.radius;
+        console.log(`The circumference is ${this.circumference.toFixed(1)}`);
+    }
+
+    getArea() {
+        this.area = Math.PI * Math.pow(this.radius, 2);
```

Comment:
same here. Also don't use `Math.pow` for squaring things, it's very inefficient. `this.radius*this.radius` works much better. Also, probably better to do the console log AFTER you return, not inside the function. The function should just give you the result.